.github/workflows/feature-branch.yml

```
@@ -0,0 +1,99 @@
+name: Feature branch CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - '**'
+      - '!main'
+      - '!adminV2'
+
+
+env:
+  NODE_VERSION: 20.x
+
+jobs:
+  code-style:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
```

It's much easier to maintain (update node version) if we:
1. setup [engine.node](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) in package.json
2. use  node-version-file instead of node-version
```suggestion
          node-version-file: "package.json"
```

---

Sure. Fixed.